server/server.go

```diff
@@ -164,6 +165,44 @@ func (s *Server) withRecovery(next http.Handler) http.Handler {
 	})
 }
 
+func (s *Server) withValidation(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		start := time.Now()
+		w = newWrappedWriter(w)
+
+		defer func() {
+			elapsed := float64(time.Since(start)) / float64(time.Second)
+			statusCode := strconv.Itoa(w.(*responseWriterWrapper).StatusCode())
+			s.Metrics.ObserveHTTPRequestDuration(r.Method, "/pr_event", statusCode, elapsed)
```

Comment:
How about using `r.URL.Path` (not sure if its the best way to get the URL basepath) instead of hardcoding `/pr_event`?

This could end up being missed when we split up the routes for different webhook events.

Reply:
Makes sense 👍